pkg/driver/driver_alicloud.go

```diff
@@ -112,16 +57,16 @@ func (c *AlicloudDriver) Create() (string, string, error) {
 		return "", "", err
 	}
 
-	request := c.createRunInstancesRequest()
+	request := ecs.CreateRunInstancesRequest()
 	//request.DryRun = requests.NewBoolean(true)
 
-	request.ImageName = c.AlicloudMachineClass.Spec.ImageName
+	request.ImageId = c.AlicloudMachineClass.Spec.ImageName
 	request.InstanceType = c.AlicloudMachineClass.Spec.InstanceType
 	request.RegionId = c.AlicloudMachineClass.Spec.Region
-	request.Zone = c.AlicloudMachineClass.Spec.Zone
-	request.SecurityGroupID = c.AlicloudMachineClass.Spec.SecurityGroupID
-	request.VSwitchID = c.AlicloudMachineClass.Spec.VSwitchID
-	request.PrivateIPAddress = c.AlicloudMachineClass.Spec.PrivateIPAddress
+	request.ZoneId = c.AlicloudMachineClass.Spec.Zone
```

Comment:
We should rename our MachineAPI fields from Zone to ZoneID also to maintain consistency